api/libproject/gcloudcontext.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "../../libproject/utils/common"
+require_relative "environments"
+require "json"
+
+class GcloudContextV2
```

Comment:
By contrast, this class just needs to grab some information (and store it for convenient retrieval). It doesn't need a run block since it doesn't need to clean anything up.